Source/WebGPU/WebGPU/CommandEncoder.h

```diff
@@ -111,6 +111,7 @@ class CommandEncoder : public CommandsMixin, public RefCountedAndCanMakeWeakPtr<
     void setEncoderState(EncoderState state) { m_state = state; }
     EncoderState getEncoderState() { return m_state; }
     NSString * encoderStateNameWrapper() { return encoderStateName(); }
+    static id<MTLFunction> mtlFunction(id<MTLDevice>);
```

Comment:
Can we name this `nopVertexFunction` or similar please?

Reply:
definitely. fixing.